src/locales/Messages.ts

```diff
@@ -429,9 +429,9 @@ export default defineMessages({
   tryThisFeatureInBeta: {
     id: 'tryThisFeatureInBeta',
     description:
-      'Try this feature in our Beta Environment on cloud.redhat.com/beta. The Beta Environment allows you to interact with new features in an active development space. Because beta pre-release software is still being developed, you may encounter bugs or flaws in availability, stability, data, or performance.',
+      'Try this feature in our Preview Environment on cloud.redhat.com/preview. The Preview Environment allows you to interact with new features in an active development space. Because beta pre-release software is still being developed, you may encounter bugs or flaws in availability, stability, data, or performance.',
```

Comment:
We should probably go ahead and update the `cloud.redhat.com` to `console.redhat.com` while we're here

Reply:
I see. Do you want me to go through all the text and change the beta to preview in all places? This was kind of random as I was searching for /beta string.